thoth/storages/graph/postgres.py

```diff
@@ -1194,6 +1197,119 @@ def create_advised_software_stack_pipfile(
 
     def sync_inspection_result(self, document) -> None:
         """Sync the given inspection document into the graph database."""
+        # Check if we have such performance model before creating any other records.
+        performance_indicator = None
+        inspection_document_id = InspectionResultsStore.get_document_id(document)
+
+        try:
+            with self._session.begin(subtransactions=True):
+
+                build_cpu = OpenShift.parse_cpu_spec(document["specification"]["build"]["requests"]["cpu"])
+                build_memory = OpenShift.parse_memory_spec(document["specification"]["build"]["requests"]["memory"])
+                run_cpu = OpenShift.parse_cpu_spec(document["specification"]["run"]["requests"]["cpu"])
+                run_memory = OpenShift.parse_memory_spec(document["specification"]["run"]["requests"]["memory"])
+
+                # Convert bytes to GiB, we need float number given the fixed int size.
+                run_memory = run_memory / (1024 ** 3)
+                build_memory = build_memory / (1024 ** 3)
+
+                run_hardware_information, run_software_environment = self._runtime_environment_conf2models(
+                    document["specification"]["run"].get("requests", {}), software_environment_type="RUN", is_user=False
```

Comment:
Reading this - we should improve our inspections are report runtime environment - run and build environments in the JSON output.